cellprofiler/modules/__init__.py

```diff
@@ -96,34 +94,34 @@
                         'morphologicalskeleton': 'MorphologicalSkeleton',
                         'namesandtypes': 'NamesAndTypes',
                         'opening': 'Opening',
-                        'noisereduction': 'NoiseReduction',
                         "overlayobjects": "OverlayObjects",
                         'overlayoutlines': 'OverlayOutlines',
+                        'reducenoise': "ReduceNoise",
                         'relateobjects': 'RelateObjects',
                         'removeholes': 'RemoveHoles',
-                        'removeobjects': 'RemoveObjects',
+                        'removeobjectsbysize': 'RemoveObjectsBySize',
                         'rescaleintensity': 'RescaleIntensity',
                         'resize': 'Resize',
                         "resizeobjects": "ResizeObjects",
+                        "savecroppedobjects": "SaveCroppedObjects",
                         'saveimages': 'SaveImages',
                         'smooth': 'Smooth',
                         'splitormergeobjects': 'SplitOrMergeObjects',
                         'straightenworms': 'StraightenWorms',
                         'matchtemplate': 'MatchTemplate',
+                        'threshold': 'Threshold',
                         'trackobjects': 'TrackObjects',
                         'tile': 'Tile',
                         'tophattransform': 'TopHatTransform',
-                        'calculateimageoverlap': 'CalculateImageOverlap',
+                        'calculateimageoverlap': 'MeasureImageOverlap',
```

Comment:
This line is a duplicate, it can be removed.

Reply:
Let's double check that "calculateimageoverlap" is not defined in `builtin_modules`, too.